README.md

```diff
@@ -1,111 +1,48 @@
-# co-log
+# co-log-polysemy
 
 ![Co-logo](https://user-images.githubusercontent.com/8126674/80955687-92f21a80-8df7-11ea-90d3-422dafdc8391.png)
 
 [![GitHub CI](https://github.com/kowainik/co-log/workflows/CI/badge.svg)](https://github.com/kowainik/co-log/actions)
-[![Build status](https://img.shields.io/travis/kowainik/co-log.svg?logo=travis)](https://travis-ci.org/kowainik/co-log)
-[![Windows build](https://ci.appveyor.com/api/projects/status/github/kowainik/co-log?branch=main&svg=true)](https://ci.appveyor.com/project/kowainik/co-log)
+[![Hackage][hk-img-ps]][hk-ps]
 [![MPL-2.0 license](https://img.shields.io/badge/license-MPL--2.0-blue.svg)](https://github.com/kowainik/co-log/blob/main/LICENSE)
 
+`co-log-polysemy` package is a implementation of the 
+[co-log-core](http://hackage.haskell.org/package/co-log-core) logging based on 
+the [polysemy](http://hackage.haskell.org/package/polysemy) extensible 
+effects library.
+
+## Goals
+
+We created the `co-log-polysemy` project in pursuit of the following goals:
+
+## Features
+
+## How to use
+
+`co-log-polysemy` is compatible with the GHC compiler
+versions starting from `8.4.4` upto `9.0.1`.
```

Comment:
I'm thinking how to remove specific versions from here as well 🤔 
Maybe just give a link to this page that shows all supported GHC versions?

* https://matrix.hackage.haskell.org/#/package/co-log-polysemy